external/fv3fit/fv3fit/pytorch/cyclegan/reloadable.py

```diff
@@ -85,11 +85,15 @@ def load(cls, path: str) -> "CycleGAN":
         """Load a serialized model from a directory."""
         return _load_pytorch(cls, path)
 
+    def to(self, device) -> "CycleGAN":
```

Comment:
I know this is torch nomeclature but really wish they used a more informative function name given that it can do a lot of things. At least whatever is passed to `device` should make the intent clearer.